jmbitcoin/setup.py

```diff
@@ -9,5 +9,5 @@
       author_email='',
       license='GPL',
       packages=['jmbitcoin'],
-      install_requires=['future', 'coincurve', 'urldecode'],
+      install_requires=['coincurve', 'python-bitcointx>=1.0.5', 'pyaes', 'urldecode'],
```

Comment:
Add python_requires='>=3.6' here?

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536/commits/0984273e60b52bf8e6d1afcada2b175f5afbb4d8 for all the packages